src/plugins/maps_legacy/public/map/decode_geo_hash.ts

```diff
@@ -17,58 +17,6 @@
  * under the License.
  */
 
-interface DecodedGeoHash {
-  latitude: number[];
-  longitude: number[];
-}
-
-/**
- * Decodes geohash to object containing
- * top-left and bottom-right corners of
- * rectangle and center point.
- */
-export function decodeGeoHash(geohash: string): DecodedGeoHash {
-  const BITS: number[] = [16, 8, 4, 2, 1];
-  const BASE32: string = '0123456789bcdefghjkmnpqrstuvwxyz';
-  let isEven: boolean = true;
-  const lat: number[] = [];
-  const lon: number[] = [];
-  lat[0] = -90.0;
-  lat[1] = 90.0;
-  lon[0] = -180.0;
-  lon[1] = 180.0;
-  let latErr: number = 90.0;
-  let lonErr: number = 180.0;
-  [...geohash].forEach((nextChar: string) => {
-    const cd: number = BASE32.indexOf(nextChar);
-    for (let j = 0; j < 5; j++) {
-      const mask: number = BITS[j];
-      if (isEven) {
-        lonErr = lonErr /= 2;
-        refineInterval(lon, cd, mask);
-      } else {
-        latErr = latErr /= 2;
-        refineInterval(lat, cd, mask);
-      }
-      isEven = !isEven;
-    }
-  });
-  lat[2] = (lat[0] + lat[1]) / 2;
-  lon[2] = (lon[0] + lon[1]) / 2;
-  return {
-    latitude: lat,
-    longitude: lon,
-  } as DecodedGeoHash;
-}
-
-function refineInterval(interval: number[], cd: number, mask: number) {
-  if (cd & mask) { /* eslint-disable-line */
-    interval[0] = (interval[0] + interval[1]) / 2;
-  } else {
-    interval[1] = (interval[0] + interval[1]) / 2;
-  }
-}
-
 export function geohashColumns(precision: number): number {
```

Comment:
Maybe we should rename this file since it no longer contains decode logic. Maybe just `geohash_columns.ts`?

Reply:
cool, done 👍